.eslintrc.js

```diff
@@ -3,7 +3,6 @@ module.exports = {
   extends: [
     "eslint:recommended",
     "plugin:vue/recommended",
-    "plugin:prettier/recommended",
```

Comment:
Upgraded `prettier` v2 has some breaking changes comparing to v1. I didn't want to add visual noise with new formatting to this PR, so I disabled it for now. I'll update the formatting in the next PR